client/src/util/stateManager/colorHelpers.ts

```diff
@@ -87,9 +87,9 @@ export function createColorQuery(
 }
 
 function _defaultColors(nObs: number): Colors {
-  const defaultCellColor = parseRGB(globals.defaultCellColor);
+  // const defaultCellColor = parseRGB(globals.defaultCellColor);
   return {
-    rgb: new Array(nObs).fill(defaultCellColor),
+    rgb: new Float32Array(3 * nObs).fill(0),
```

Comment:
this breaks the global defaultCellColor config. Suggest maintaining that, OR if you find the global not useful, cleaning it up (the comment on line 90, and the line in globals.ts)

Reply:
Ah shoot, I forgot to fix this.